package.json

```diff
@@ -76,7 +76,7 @@
     "start": "node server.js",
     "test:db:migrate": "knex migrate:latest --knexfile db/knexfile.js --env tests",
     "test:db:seed": "knex seed:run --knexfile db/knexfile.js --env tests",
```

Comment:
May not need this script now that `resetDB` is doing the seed.

Reply:
I think the only difference is that the resetDB deletes the seeded data after the tests run (I think).